pals/alex.md

```diff
@@ -0,0 +1,68 @@
+# Alex Gozin #
```

Comment:
I'm afraid the name must be nixed - we already have two players with Alex in or around one of their names.